runtime_scan/pkg/provider/cloudinit/cloud-init.tmpl.yaml

```diff
@@ -21,6 +21,7 @@ write_files:
       ExecStartPre=mkdir -p /var/opt/vmclarity
       ExecStartPre=docker pull {{ .ScannerImage }}
       ExecStart=docker run --rm --name %n --privileged \
+          -v /:/hostfs \
```

Comment:
do we still need to mount /run and /dev explicitly with that change?

Reply:
yeah we need to keep those mounts, those directories are being mounted to the container's /run and /dev to allow the containers tools to operate against them. This change is mounting the rest of the root fs into /hostfs which doesn't affect the containers operation at all, but it just makes the container aware of all the directories